sql/moz-fx-data-shared-prod/telemetry_derived/addons_daily_v1/schema.yaml

```diff
@@ -0,0 +1,187 @@
+fields:
```

Comment:
I think that adding this schema file is an unrelated change. Can we remove this?